queries/ocaml/rainbow-blocks.scm

```diff
@@ -2,13 +2,35 @@
 (let_expression ; Line 170
   (value_definition
     "let" @delimiter
-    ("rec" @delimiter)?)
+    ("rec" @delimiter)?
+    ("and" @delimiter (let_binding))*)
```

Comment:
Do we really need the `(let_binding)` after the `"and"`?

Reply:
I see. We could use the wildcard `_` (or `(_)` if we only want to match named nodes) instead of `(let_binding)`. Would that make more sense or would it be too generic?